docs/deployment_policy.md

```diff
@@ -34,12 +34,19 @@ Following are the fields in the JSON representation of a deployment policy:
   - `inputs`: A list of service variables to set.
     - `name`: The name of the variable. This is the same as a variable name found in `userInputs` as defined [here](./service_def.md).
     - `value`: The value to be assigned to the variable. Service variables are typed as described in `userInputs` defined [here](./service_def.md).
+- `secretBinding`: This section is used to bind secret names defined in the servcie with the secret names in the vault. The secret value will be retrived from the vault and passed to the service container at the deployment time. The secret value is used by the service container to access other applications.
```

Comment:
typo servcie
replace references to vault with secret provider

Reply:
done